config/application.rb

```diff
@@ -23,5 +24,6 @@ class Application < Rails::Application
     # -- all .rb files in that directory are automatically loaded.
 
     config.active_record.schema_format = :sql
+    config.time_zone = "Europe/Bratislava"
```

Comment:
Toto nie je dobry napad, my chceme aby app robila v UTC a ak chceme ine formaty, tak to uvedieme explicitne. Tvoja skusenost je aka?